tutorials/Tutorial5_Evaluation.py

```diff
@@ -162,6 +163,8 @@ def tutorial5_evaluation():
         reader.print_time()
         print()
         eval_reader.print(mode="pipeline")
+    else:
+        raise Exception()
```

Comment:
Let's make this something like `ValueError(f'style={style} is not a valid option.')`